app/config.js

```diff
@@ -272,11 +272,11 @@ settings.downloadFormatParamsPhysical = {
 
 // Korp backend URL
 // Always use the backend on the production server
-var korpBackendServer = "https://www.kielipankki.fi";
-settings.korpBackendURL = korpBackendServer + "/korp/api8";
+// var korpBackendServer = "https://www.kielipankki.fi";
+// settings.korpBackendURL = korpBackendServer + "/korp/api8";
 // // Alternatively, use the backend on the same site as the frontend
 // settings.korpBackendURL =
```

Comment:
I think this line should also be uncommented for the change to work.

Reply:
In general, I suppose this is a change required for Korp test instances to work in the new environment.

Reply:
Oh right, of course! Thanks.